old Makefile

```diff
@@ -0,0 +1,53 @@
+MODE = release
```

Comment:
is this kept for backup ?

Reply:
Yes, removed

Reply:
Still exists? Do you intend to delete it ?